bedrock/base/templates/includes/global-nav.html

```diff
@@ -211,6 +211,11 @@ <h3 class="summary" data-id="technology">
                   </a>
                 </li>
                 <li>
+                  <a href="{{ url('mozorg.developer') }}" data-link-type="nav" data-link-name="Mozilla Developers" data-link-group="Technology">
+                    {{ _('Mozilla Developers') }}
```

Comment:
I agree. This page doesn't really fit anywhere, it's kind of a standalone landing page. It shouldn't be hard to add a subnav to the technology page... I'll try it.